pipelines/matrix/src/matrix/pipelines/matrix_generation/multi_model_pipeline.py

```diff
@@ -0,0 +1,125 @@
+from kedro.pipeline import Pipeline, pipeline
```

Comment:
I don't think I agree with this pipeline being multi model, I thought the idea was for the modelling pipeline to still output a _single_ model

dynamic fan outs within pipelines are usually best if they converge on a single dataset to handover into the next pipeline

Reply:
Should the modelling pipeline then not create the `ModelWrapper` from each of the models, and matrix generation then still get a single model as before? or am I missing something